consensus/consensus_tests/pacemaker_test.go

```diff
@@ -133,7 +133,7 @@ func TestPacemakerCatchupSameStepDifferentRounds(t *testing.T) {
 		Hash:              hex.EncodeToString(appHash),
 		NumTxs:            0,
 		LastBlockHash:     "",
-		ProposerAddress:   leader.Address.Bytes(),
+		ProposerAddress:   leader.GetP2PAddress().Bytes(),
```

Comment:
The `P2PAddress` should not be the address contained in the block header as it's just an ephemeral P2P identity. It should be the `Consensus Address`

Reply:
This should fix it

https://github.com/pokt-network/pocket/pull/235/files#diff-c474ba0a5fb5b813e523913230483776eb1344705fb79db6d267ced0566c85f8R132-R141